### DISCUSSION TITLE
Frigate ccript - configuring Frigate to store recording/snapshots on NAS

### DISCUSSION THREAD

**Discussion by jnrollins** [community-scripts/ProxmoxVE#959] [Q&A] [answered]
I am have been working on getting Frigate to store data on my NAS and have hit a weird speedbump. Here is where I am at - samba share mounted to pve, folder from pve mounted/passed through to frigate lxc, folder is accessible in lsc and can see folders on NAS, mount point/mapping is modified in docker-compose.yml to /mnt/pve/frigate_media:/media/dfrigate. As I understand things that is all I needed to do. However...when I change the mapping in docker-compose.yml to /mnt/pve/frigate_media:/media/frigate....nothing. Nothing gets saved to the NAS. I even renamed the existing /media/frigate to /media/frigate_old. Lo and behold when I restart the lxc /media/frigate exists again. What am I missing here? Does Frigate recreate /media/frigate on boot if it doesn't exist? 

I ask here because I am curious how the lxc installation differs from "pure docker." Most resources seem to reference a standard docker installation or a home assistant installation. Hoping someone can fill me in if there is a nuance to this setup that I am unaware of. Thanks in advance.

**Comment by xijio** [comment, marked as answer]
Try this way of mounting the shares instead: 

https://forum.proxmox.com/threads/tutorial-unprivileged-lxcs-mount-cifs-shares.101795/

It works for me!

**Reply by jnrollins** [reply]
I will look through that tonight and give it a shot. Thanks for your reply.
